src/testUtils/MockFactories.ts

```diff
@@ -0,0 +1,92 @@
+import { BigNumber } from '@polymathnetwork/abi-wrappers';
```

Comment:
This file should be named without a capital letter since it's not exporting a class